getting this error during review:
```
error: build of '/nix/store/ppcrv05yxvw27myf4wa98vq8nlsvvbr5-code-browser-7.1.20.drv' on 'ssh://nix@morpheus.r' failed: builder for '/nix/store/ppcrv05yxvw27myf4wa98vq8nlsvvbr5-code-browser-7.1.20.drv' failed with exit code 2
builder for '/nix/store/ppcrv05yxvw27myf4wa98vq8nlsvvbr5-code-browser-7.1.20.drv' failed with exit code 1; last 10 log lines:
  ar rcs /build/code-browser-7.1.20/libcopper-gtk.a /build/code-browser-7.1.20/gtk.o
  make[1]: Leaving directory '/build/code-browser-7.1.20/libs/copper-ui'
  make[1]: Entering directory '/build/code-browser-7.1.20'
  Compiling...
  Resources...
  /nix/store/npfsrhkjww5q7sax7p7ijcrj3wlbrxn7-bash-4.4-p23/bin/bash: libs/copper-ui/utils/gen-res: /usr/bin/env: bad interpreter: No such file or directory
  make[1]: *** [Makefile:62: /build/code-browser-7.1.20/resources.o] Error 126
  make[1]: Leaving directory '/build/code-browser-7.1.20'
  make: *** [Makefile:94: with-local-libs] Error 2
  builder for '/nix/store/ppcrv05yxvw27myf4wa98vq8nlsvvbr5-code-browser-7.1.20.drv' failed with exit code 2
error: build of '/nix/store/dfhn2mbswlkr8zx6zqsbdy64kd35zgxz-code-browser-7.1.20.drv' on 'ssh://nix@morpheus.r' failed: builder for '/nix/store/dfhn2mbswlkr8zx6zqsbdy64kd35zgxz-code-browser-7.1.20.drv' failed with exit code 2
builder for '/nix/store/dfhn2mbswlkr8zx6zqsbdy64kd35zgxz-code-browser-7.1.20.drv' failed with exit code 1; last 10 log lines:
  ar rcs /build/code-browser-7.1.20/libcopper-gtk.a /build/code-browser-7.1.20/gtk.o
  make[1]: Leaving directory '/build/code-browser-7.1.20/libs/copper-ui'
  make[1]: Entering directory '/build/code-browser-7.1.20'
  Compiling...
  Resources...
  /nix/store/npfsrhkjww5q7sax7p7ijcrj3wlbrxn7-bash-4.4-p23/bin/bash: libs/copper-ui/utils/gen-res: /usr/bin/env: bad interpreter: No such file or directory
  make[1]: *** [Makefile:62: /build/code-browser-7.1.20/resources.o] Error 126
  make[1]: Leaving directory '/build/code-browser-7.1.20'
  make: *** [Makefile:94: with-local-libs] Error 2
  builder for '/nix/store/dfhn2mbswlkr8zx6zqsbdy64kd35zgxz-code-browser-7.1.20.drv' failed with exit code 2
cannot build derivation '/nix/store/p7bf1r38lcjxrhrcqm60pphdj6mbsdy7-env.drv': 2 dependencies couldn't be built
[0 built (2 failed), 1 copied (0.9 MiB), 0.0 MiB DL]
error: build of '/nix/store/p7bf1r38lcjxrhrcqm60pphdj6mbsdy7-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/88548
2 packages failed to build:
code-browser-gtk code-browser-qt
```

--- a/pkgs/applications/editors/code-browser/default.nix
+++ b/pkgs/applications/editors/code-browser/default.nix
@@ -1,0 +1,48 @@
+{ stdenv
+, fetchurl
+, copper
+, ruby
+, python3
+, qtbase
+, gtk3
+, pkg-config
+, withQt ? false
+, withGtk ? false, wrapQtAppsHook ? null
+}:
+stdenv.mkDerivation rec {
+  pname = "code-browser";
+  version = "7.1.20";
+  src = fetchurl {
+    url = "https://tibleiz.net/download/code-browser-${version}-src.tar.gz";
+    sha256 = "1svi0v3h42h2lrb8c7pjvqc8019v1p20ibsnl48pfhl8d96mmdnz";
+  };
+  postPatch = ''
+    substituteInPlace Makefile --replace "LFLAGS=-no-pie" "LFLAGS=-no-pie -L."
+    substituteInPlace libs/copper-ui/Makefile --replace "moc -o" "${qtbase.dev}/bin/moc -o"
+    patchShebangs .
+  '';
+  nativeBuildInputs = [ copper
+                        python3
+                        ruby
+                        qtbase
+                        gtk3
+                        pkg-config
+                      ]
+  ++ stdenv.lib.optionals withQt [ wrapQtAppsHook ];
+  buildInputs = stdenv.lib.optionals withQt [ qtbase ]
+                ++ stdenv.lib.optionals withGtk [ gtk3 ];
+  makeFlags = [
+    "prefix=$(out)"
+    "COPPER=${copper}/bin/copper-elf64"
+    "with-local-libs"
+    "QINC=${qtbase.dev}/include"
+  ]
+  ++ stdenv.lib.optionals withQt [ "UI=qt" ]
+  ++ stdenv.lib.optionals withGtk [ "UI=gtk" ];
+  meta = with stdenv.lib; {
+    description = "Folding text editor, designed to hierarchically structure any kind of text file and especially source code.";
+    homepage = "https://tibleiz.net/code-browser/";
+    license = licenses.gpl2;
+    platforms = platforms.x86_64;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -2823,6 +2823,12 @@ in
 
   biosdevname = callPackage ../tools/networking/biosdevname { };
 
+  code-browser-qt = libsForQt5.callPackage ../applications/editors/code-browser { withQt = true;
+                                                                                };
+  code-browser-gtk = callPackage ../applications/editors/code-browser { withGtk = true;
+                                                                        qtbase = qt5.qtbase;
+                                                                      };
+
   c14 = callPackage ../applications/networking/c14 { };
 
   certstrap = callPackage ../tools/security/certstrap { };

